frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -460,3 +470,79 @@ action.GitMachete.CheckoutParentAction.undefined.branch-name=Branch (''{0}'') do
 action.GitMachete.CheckoutFirstChildAction.text=Git Machete: Checkout First Child Branch
 action.GitMachete.CheckoutFirstChildAction.description=Checkout first child branch (''{0}'')
 action.GitMachete.CheckoutFirstChildAction.undefined.branch-name=Branch (''{0}'') does not have any children in the branch layout
+
+# Traverse
+
+action.GitMachete.BaseTraverseAction.description=Traverse
+action.GitMachete.BaseTraverseAction.description.empty-layout=Cannot traverse an empty branch layout
+
+action.GitMachete.BaseTraverseAction.dialog.traverse-approval.title=Traverse Info
+action.GitMachete.BaseTraverseAction.dialog.traverse-approval.text.HTML=<html>\
+  This action triggers an interactive walk over the branches and suggests actions (pushes/pulls/rebases) \
+  necessary to achieve both child-to-parent and local-to-remote synchronization.<br><br>\
+  Do you want to proceed?\
+  </html>
+
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.text.HTML=\
+  <html>\
+    <p>The branch <b>{0}</b> is diverged from remote <b>{1}</b>, and it is <b>{2}</b> remote.</p> \
+    <p>Please choose the preferred action:</p>\
+  </html>
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.title=Resolve Divergence from Remote
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.newer-than-remote=newer than
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.older-than-remote=older than
+
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.do-not-resolve-option=Do not sync
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.do-not-resolve-option.comment=Note: the remote and local branches will remain intact; traverse will continue
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.force-push-option=Force push
+action.GitMachete.BaseTraverseAction.dialog.diverged-from-remote.force-push.comment=Warning: you will lose the remote commits
```

Comment:
Maybe: _the remote commits will no longer be easily available (you might need to resort to reflogs to retrieve them)_? 🤔

Reply:
tbh it sounds so nerdish that I would feel better simply seeing "lose" Xd

Reply:
Okay :trollface:

Reply:
a person who knows that can _resort to reflogs to retrieve them_ will obviously understand that our plugin uses git under the hood (what else could it use?) and in git nothing is easily loosable

Reply:
Maybe... `might lose`? 🙏🏻 just **not** to make it too alarming

Reply:
If I were a git newbie I would rather see `lose` than `might lose` b/c I will not know how to retrieve them

my thinking is like this
- who knows - knows that will not lose even though it says `lose`, and the message wont be alarming  for them
- who does not know - will lose them actually, and the message should be alarming for them

Reply:
what's _the consistency to the cli argument_?

Reply:
![image](https://user-images.githubusercontent.com/19799111/209221214-5d6832af-6f24-4428-b0d5-4df13825faf8.png)
 I see - simply name the git action and do not warn 😆  that's an option

Reply:
no warn for the reset to...
![image](https://user-images.githubusercontent.com/19799111/209221852-a588caa7-e28b-43a8-bab6-43e584f5a2b3.png)

Reply:
wdyt about removing the warnings? we will meet halfway and the message will be correct

Reply:
Okay, let's remove them then!